harvester/records/record.py

```diff
@@ -386,38 +389,63 @@ def _gbl_mdVersion_s(self) -> str:
     def _dct_references_s(self) -> str:
         """Shared field method: dct_references_s
 
-        Builds http URLs for the data zip file (if MIT harvest), source and normalized
-        metadata files, in the S3 CDN bucket.
+        Builds a JSON string payload of links for the record.  Work is offloaded to
+        methods specific to MIT and OGM harvests.
         """
-        # determine CDN public or restricted folder
-        cdn_folder = {True: "restricted", False: "public"}[self.is_restricted]
+        if self.origin == "ogm":
+            urls_dict = self._dct_references_s_ogm()
+        elif self.origin == "mit":
+            urls_dict = self._dct_references_s_mit()
+        else:
+            message = f"Source not recognized: {self.origin}"
+            raise ValueError(message)
+        return json.dumps(urls_dict)
+
+    def _dct_references_s_mit(self) -> dict:
+        """Create dct_references_s JSON string for MIT harvests.
 
-        # build URLs payload
+        For MIT harvests, this includes the data zip file, source and normalized metadata
```

Comment:
Update `normalized` to `Aardvark` given that the label has changed?

Reply:
Appreciate the suggestion, but I'd propose keeping as "normalized".  My reasoning is this: while the _label_ in the output is "Aardvark" now indicating it's actual type, in the context of the harvester (e.g. in code) I think "normalized" is still widely used to refer to it, differentiating from "source", but not to be confused with a possible "source" type of Aardvark once OGM work hits.

Reply:
Makes sense, withdrawn!